### ISSUE TITLE
Active formatting bar entries do not meet the contrast requirements 

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/text#2828]
As discussed with @jancborchardt we either need the dot/bar indicator back or need to increase the contrast by using a darker color for the icon in active mode.

Active should become a dedicated state of the vue button competent.

**Comment by juliushaertl** [comment]
Closing as a duplicate of https://github.com/nextcloud/text/issues/3860 as this has screenshots
